components/style/animation.rs

```diff
@@ -367,7 +372,10 @@ impl PropertyAnimation {
                     out = 1.0;
                 }
                 out
-            }
+            },
+            TransitionTimingFunction::Keyword(_) => {
+                unreachable!("Keyword function should not appear")
```

Comment:
You probably want `panic!` here, `unreachable!` AFAIK is for more obvious code, not for invariants maintained elsewhere.